src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
```

Comment:
How about to use @Rule to make tests for exceptions? You use different exception messages in your code so mey be its better to check it?

Not sure that is mistake but...
In many cases:
        inputData = List.of("type,fruit,quantity",
                "b,1banana,20",
                "b,2apple,100",
                "s,44444banana,100",
                " p,banana,13");
What will happen if your class will throw exception at  "b,1banana,20" lina but will pass "s,44444banana,100"?
Isn't better to use always right List and set just 1 disright line to it?

Reply:
About @Rule - thanks for advice